Solutions/Lookout Cloud Security/Data Connectors/LookoutCSConnector/azuredeploy_LookoutCloudSecurityConnector_FunctionApp.json

```diff
@@ -0,0 +1,243 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "FunctionName": {
+            "defaultValue": "fnLoCSCon",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 11
+        },
+        "LookoutClientId": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The unique ID that was provided for this client"
+            }
+        },
+        "LookoutApiSecret": {
+            "type": "securestring",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The secret that was provided for this client"
+            }
+        },
+        "WorkspaceID": {
+            "type": "string",
+            "defaultValue": "<workspaceID>"
+        },
+        "WorkspaceKey": {
+            "type": "securestring",
+            "defaultValue": "<workspaceKey>"
+        },
+		 "Baseurl": {
+            "type": "string",
+            "metadata":{
+            "description": "The URL used to access this client"
```

Comment:
This can be more descriptive and some example

Reply:
As discussed provide instruction that the url must start with https://